Algorithms/GRAPHS/Floyd Warshall/floydwarshall.cpp

```diff
@@ -0,0 +1,69 @@
+#include<iostream>
```

Comment:
Floyd Warshall is already present in the Minimum spanning tree folder in graphs.